front/src/common/redux/auth/api/auth.api.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+
+async function loginApi(request) {
+    const url = 'http://127.0.0.1:5000/auth/login';
```

Comment:
Please, use the environment variable here

Reply:
done